spec/Dockerfile.puppetdb

```diff
@@ -0,0 +1,22 @@
+FROM puppet/puppetdb
+
+ARG hostname="bolt-puppetdb"
+
+#ENV PUPPETSERVER_HOSTNAME "$hostname"
```

Comment:
Is there a reason this is commented? I don't see harm in setting it and it looks like it'll be useful for https://github.com/puppetlabs/puppetdb/pull/2738#discussion_r239637958